reify.go

```diff
@@ -17,7 +17,10 @@ func (c *Config) Unpack(to interface{}, options ...Option) error {
 	}
 
 	vTo := reflect.ValueOf(to)
-	if to == nil || (vTo.Kind() != reflect.Ptr && vTo.Kind() != reflect.Map) {
+
+	k := vTo.Kind()
+	isValid := to != nil && (k == reflect.Ptr || k == reflect.Map)
```

Comment:
I think it's easier to read then before. it says: to must not be nil and must be either a pointer or a map